root/edit/details/EditArtist.js

```diff
@@ -0,0 +1,180 @@
+  artistEndLabel,
+} from '../../artist/utils';
+import Diff from '../../static/scripts/edit/components/edit/Diff';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import FullChangeDiff from '../../static/scripts/edit/components/edit/FullChangeDiff';
+import DescriptiveLink from '../../static/scripts/common/components/DescriptiveLink';
+import formatDate from '../../static/scripts/common/utility/formatDate';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+import {commaOnlyListText} from '../../static/scripts/common/i18n/commaOnlyList';
+
+type EditArtistProps = {
+  ...EditT,
+  +display_data: {
+    +area?: CompT<AreaT | null>,
+    +artist: ArtistT,
+    +begin_area?: CompT<AreaT | null>,
+    +begin_date?: CompT<PartialDateT>,
+    +comment?: CompT<string>,
+    +end_area?: CompT<AreaT | null>,
+    +end_date?: CompT<PartialDateT>,
+    +ended?: CompT<boolean>,
+    +gender?: CompT<GenderT | null>,
+    +ipi_codes?: CompT<$ReadOnlyArray<string> | null>,
+    +isni_codes?: CompT<$ReadOnlyArray<string> | null>,
+    +name: CompT<string>,
+    +sort_name?: CompT<string>,
+    +type?: CompT<ArtistTypeT>,
+  },
+}
+
+const EditArtist = ({edit}: {edit: EditArtistProps}) => {
+  const display = edit.display_data;
+  const area = display.area;
+  const beginDate = display.begin_date;
+  const beginArea = display.begin_area;
+  const endDate = display.end_date;
+  const endArea = display.end_area;
+  const ended = display.ended;
+  const gender = display.gender;
+  const type = display.type;
+  let displayTypeId;
+  if (type && (type.new !== type.old)) {
+    displayTypeId = 0;
+  } else {
+    displayTypeId = display.artist.typeID;
+  }
+  return (
+    <>
+      <table className="details">
+        <tbody>
+          <tr>
+            <th>{l('Artist:')}</th>
+            <td><EntityLink entity={display.artist} /></td>
+          </tr>
+        </tbody>
+      </table>
+      <table className="details edit-artist">
+        <tbody>
+          {display.name ? (
+            <Diff
+              label={addColonText(l('Name'))}
+              newText={display.name.new}
+              oldText={display.name.old}
+              split="\s+"
+            />
+          ) : null}
+          {display.sort_name ? (
+            <Diff
+              label={addColonText(l('Sort name'))}
+              newText={display.sort_name?.new ?? ""}
+              oldText={display.sort_name?.old ?? ""}
+              split="\s+"
+            />
+          ) : null}
+          {display.comment ? (
+            <Diff
+              label={addColonText(l('Disambiguation'))}
+              newText={display.comment?.new ?? ""}
+              oldText={display.comment?.old ?? ""}
+              split="\s+"
+            />
+          ) : null}
+          {type ? (
+            <FullChangeDiff
+              label={addColonText(l('Type'))}
+              newContent={type.new.name}
+              oldContent={type.old.name}
```

Comment:
These should probably be translated (lp_attributes with 'artist_type'). 

Do old and new always exist and are defined? (I see elsewhere you check for `new?.` but not here)